File: CS/TestMyRadioGroup/MyRadioGroupItemCollection.cs
using System;
using System.ComponentModel;
using DevExpress.XtraEditors.Controls;



namespace TestMyRadioGroup
{
    [ListBindable(false)]
    public class MyRadioGroupItemCollection : RadioGroupItemCollection
    {
        public new MyRadioGroupItem this[int index]
        {
            get { return List[index] as MyRadioGroupItem; }
            set { List[index] = value; }
        }


        public MyRadioGroupItemCollection()
            : base()
        {
        }
   }
}

File: VB/TestMyRadioGroup/MyRadioGroupItemCollection.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel
Imports DevExpress.XtraEditors.Controls



Namespace TestMyRadioGroup
	<ListBindable(False)>
	Public Class MyRadioGroupItemCollection
		Inherits RadioGroupItemCollection
		Default Public Shadows Property Item(ByVal index As Integer) As MyRadioGroupItem
			Get
				Return TryCast(List(index), MyRadioGroupItem)
			End Get
			Set(ByVal value As MyRadioGroupItem)
				List(index) = value
			End Set
		End Property


		Public Sub New()
			MyBase.New()
		End Sub
	End Class
End Namespace
